as/cNFT/assembly/market.ts

```diff
@@ -78,16 +124,65 @@ export function remove_bid(tokenId: string): void {
     logging.log(log)
 }
 
+
+
+/**
+ * Get bids for a particular token.
+ *
+ *
+ * **Basic usage example:**
+ *
+ * Assume we need to get all the bids that set to the token with token id = `jenny911038`,
+ * ```
+ * const bids_for_the_token = get_bids("jenny911038");
+ * ```
+ *
+ * @param tokenId ID of the token that need to get the bids
+ * @return Object of the bids that the token has
+ */
 @nearBindgen
 export function get_bids(tokenId: string): BidsByBidder {
     return persistent_market.get(tokenId)
 }
 
+
+
+/**
+ * Get bids set by a particular user.
+ *
+ *
+ * **Basic usage example:**
+ *
+ * Assume we need to get all the bids that set by a user with account id = `alice.test.near`,
+ * ```
+ * const bids_by_user = get_bidder_bids("alice.test.near");
+ * ```
+ *
+ * @param accountId ID of the user that need to get the bids set by him
+ * @return Array of bids that user has
+ */
 @nearBindgen
 export function get_bidder_bids(accountId: string): Bid[] {
     return persistent_market.get_by_bidder(accountId)
 }
 
+
+
+/**
+ * Accept a certain bid that owner would like.
```

Comment:
`that owner would like.` is not necessary here
maybe replcae it with `on a particular token`